### PR TITLE
Added how to setup SVN access via SSH keys in 16-using-svn.md

### PR DESCRIPTION
**Pull request by elgicse**
Added how to setup SVN access via SSH keys.
Removed the '$' in front of bash commands (not used in any other lesson).

### CODE REVIEW

File: 16-using-svn.md [hunk 1]
```diff
@@ -149,5 +149,25 @@ This will open your editor, where you should enter a message describing the fold
 You can checkout this repository and begin playing with it using the commands above.
 
 ```bash
-$ svn co svn+ssh://svn.cern.ch/reps/lhcbdocs/Users/$USERNAME/TestFolder
+svn co svn+ssh://svn.cern.ch/reps/lhcbdocs/Users/$USERNAME/TestFolder
 ```
+
+
+> ## Using SVN outside the lxplus cluster {.callout}
+> Sometimes you may want to `getpack` and/or `commit` from other places, e.g. your local machine, the online cluster...
```
Comment (betatim) [inline 16-using-svn.md]: Would change it to "If you want to getpack .... from other places". I dislike the "sometimes", "maybe", "potentially" kind of style

**Comment by betatim** [comment]
removing the `$` at the start of the line, is this consistent with the other lessons? Think I prefer it, so if it isn't consistent we should file an issue to change it everywhere.

**Comment by elgicse** [comment]
About the `$`: I had a look at 3-4 other lessons, and I did not find it in any of them.

**Comment by betatim** [comment]
Ok, we all bow to the god of small details and consistency :smile:  Looks good to me -> merged.

Thanks a lot and congratulations on your first contribution!! :gift: :rocket:

**Pull request merged**